internal/fn/fn.go

```diff
@@ -21,25 +21,23 @@ import (
 	"context"
 	"fmt"
 	"log"
-	"time"
 
 	"cuelang.org/go/cue"
 	"cuelang.org/go/cue/cuecontext"
 	"cuelang.org/go/cue/parser"
 
+	input "github.com/crossplane-contrib/function-cue/input/v1beta1"
 	"github.com/crossplane/crossplane-runtime/pkg/logging"
 	"github.com/crossplane/function-sdk-go"
 	fnv1beta1 "github.com/crossplane/function-sdk-go/proto/v1beta1"
 	"github.com/crossplane/function-sdk-go/request"
 	"github.com/crossplane/function-sdk-go/response"
-	input "github.com/elastic/crossplane-function-cue/pkg/input/v1beta1"
 	"github.com/pkg/errors"
 	"google.golang.org/protobuf/encoding/protojson"
-	"google.golang.org/protobuf/types/known/durationpb"
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
-const debugAnnotation = "crossplane-function-cue/debug"
+const debugAnnotation = "function-cue/debug"
```

Comment:
Maybe `cue.fn.crossplane.io/debug`? We often use crossplane.io in annotation keys.

Reply:
Sounds good. I'll make this change. Wasn't sure what namespace to use.

Reply:
Fixed in https://github.com/crossplane-contrib/crossplane-function-cue/pull/1/commits/35341ddffcc33886aff8c5adb381f736a5724233